draft-ietf-tls-tls13.md

```diff
@@ -3662,7 +3563,7 @@ implementations SHOULD delete traffic_secret_N. Once the directional
 keys are no longer needed, they SHOULD be deleted as well.
 
 
-## Traffic Key Calculation
+## Traffic Key Calculation {#traffic-key-calculation}
```

Comment:
This is not needed; I recommend reverting this one line. All sections have automatically generated anchor links based on their names in exactly this format. Note line 946 in the removed section above where this section was already cited without issue. You only need to give it a custom anchor if the title is long and a shorter link would be preferred. (e.g. {#nonce} above is fine, though the title itself isn't that long)